### ISSUE TITLE
Support smart life light bar (RGB and front light) 

### ISSUE THREAD

**Issue by avielc** [milo526/homebridge-tuya-web#416]
Hi
I just received a really nice product - a light bar for the computer that also has RGB light for background effects
https://www.amazon.com/dp/B099Z9V3KM

**Is your feature request related to a problem? Please describe.**
This product resolves me an issue of lights on the keyboard. 
and for some reason the plugin can't turn on or off the actual light part (it only works on the RGB background lights.

**Describe the solution you'd like**
Basically integrate 2 light controllers in one (or as a grouped icon) 
so you can control either the front light bar with warm\cold on\off & the backlight RGB.

**Describe alternatives you've considered**
didn't consider anything 

**Additional context**
I ran into the whole: 
Characteristics errors with this light bar. changing it to Dimmer (as suggested) made me lost the ability to control the RGB lights.

```
[3/9/2022, 7:06:11 PM] [TuyaWebPlatform] [Light bar] Characteristic.Brightness - Characteristic 'Brightness' will receive value higher than allowed (110) since provided Tuya value (110) exceeds configured maximum Tuya value (100). Please update your configuration!
[3/9/2022, 7:06:11 PM] [@milo526/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 110 exceeded maximum of 100. See https://git.io/JtMGR for more info.
[3/9/2022, 7:06:11 PM] [TuyaWebPlatform] [Light bar] Characteristic.ColorTemperature - Characteristic 'ColorTemperature' will receive value lower than allowed mired (-902) since provided Tuya kelvin value (25035) exceeds configured maximum Tuya kelvin value (2700). Please update your configuration!
[3/9/2022, 7:06:11 PM] [@milo526/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Color Temperature': characteristic was supplied illegal value: number -902 exceeded minimum of 153.84615384615384. See https://git.io/JtMGR for more info.
```

**Comment by avielc** [comment]
Hi, 
This is definitely not a stale - I'm really intrested to know if it's possible to allow the light bar be exposed to the HomeKit\Homebridge part. 

Thank you!

**Comment by avielc** [comment]
Not stale

**Comment by avielc** [comment]
@milo526  - Hi mate, any chance to get your thoughts on this one please?
Thank you!

**Comment by milo526** [comment]
Could you try using the manufacturers app to switch the light bar into white mode, from there you might be able to adjust colortemperature and brightness.

**Comment by avielc** [comment]
Thanks for replying!
The thing is, there are 2 light bars on it, a frontal white\warm\cold light and a backlight rgb that supposed to shine on the wall as environmental feat. 
for some reason only the backlight is available (called Dimmer\Mode)
Adding a picture of the device menu (look at the bottom for all the options available)
![Screenshot 2022-03-30 at 10 33 38](https://user-images.githubusercontent.com/13572969/160777052-d3213868-e4aa-4031-88da-d2ae2b1fe207.png)

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Comment by avielc** [comment]
got it, I'll look it up (probably can only do it in a few hours as I'm out of home now)

**Comment by avielc** [comment]
@milo526 
sorry for the very late reply (got swamped at work)
finally sat down to run that script. 
Here are the results, it appeared there:
`{   'data': {   'brightness': '400',
                                                  'color_mode': 'white',
                                                  'color_temp': 18647,
                                                  'online': True,
                                                  'state': 'false'},
                                      'dev_type': 'light',
                                      'ha_type': 'light',
                                      'icon': 'https://images.tuyaeu.com/smart/icon/ay1498203399759ASLsi/1625291059108c27215ab.jpg',
                                      'id': 'bf612889cd547a47b6rcwj',
                                      'name': 'Light bar'}`
let me know what else you need. 
Basically as mentioned this light bar has rgb backlight (which is exposed in the Homekit) and the front light (cold\warm\daylight) which isn't. I would really love the fron one to be exposed (much more needed than the RGB part) if both can be, that's even better

Thanks again!

**Comment by milo526** [comment]
Could you switch it through the Tuya app (or whatever first party app you are using) to color mode and run the same thing again?
It seems like switching the color-mode switches the exposed device as we can clearly only see the color temperature (white tones) right now.

**Comment by avielc** [comment]
If you look at the screenshot I added above. 
when opening the device via smart life you see “White” which is the front light and “Dimmer” which is the RGB backlight. 
only the RGB is exposed. 
not sure what you meant by switching it through

and yes only color mode is available. Can we expose the other light?

**Comment by milo526** [comment]
Okay, that also solves it. There is only one devices exposed via the API thus this plugin will sadly not be able to help you, please take a look at Tuya's official homebridge plugin, that one supports more features since it uses a different API.

**Comment by milo526** [comment]
Yeah I know, I wish it'd be different but Tuya has made their stance very clear, they will not expand the API that this plugin is using :/ so sadly there is nothing else I can do for you.

**Comment by avielc** [comment]
No worries.  Thanks for helping this far. Really appreciate it mate, keep up the plug-in it’s awesome!
